.github/workflows/cuda-ci.yml

```diff
@@ -15,10 +15,10 @@ jobs:
     runs-on: "ubuntu-latest"
     name: Build wheel for Pull Request
     steps:
-      - uses: actions/checkout@v4
+      - uses: actions/checkout@v5
 
       - name: Build wheels
-        uses: pypa/cibuildwheel@9e4e50bd76b3190f55304387e333f6234823ea9b
+        uses: pypa/cibuildwheel@7c619efba910c04005a835b110b057fc28fd6e93
```

Comment:
While you are at it it would be nice to reinstate the version comment (IIRC for some reason it was not updated by dependabot and we removed it to avoid the comment being misleading) ... dependabot seems to work fine for `pypa/gh-action-pypi-publish` below.


```suggestion
        uses: pypa/cibuildwheel@7c619efba910c04005a835b110b057fc28fd6e93 # v3.2.0
```

This is the commit that points to v3.2.0 https://github.com/pypa/cibuildwheel/commit/7c619efba910c04005a835b110b057fc28fd6e93